pkg/securitypolicy/framework.rego

```diff
@@ -266,17 +266,147 @@ container_started {
     data.metadata.started[input.containerID]
 }
 
+capsList_ok(allowed_caps_list, requested_caps_list) {
+    count(allowed_caps_list) == count(requested_caps_list)
+
+    every cap in requested_caps_list {
+        some allowed in allowed_caps_list
+        cap == allowed
+    }
+
+    every allowed in allowed_caps_list {
+        some cap in requested_caps_list
+        allowed == cap
+    }
+}
+
+filter_capsList_by_allowed(allowed_caps_list, requested_caps_list) := caps {
+    # find a subset of the capabilities that are valid
+    caps := {cap |
```

Comment:
this mirrors what we do in env rules. I intentionally made them be as similar as possible. @matajoh is the best expert in this area as he did that work.

Reply:
so these are lists. they aren't sets, it is creating a set. i think you are suggesting creating two sets and then doing an intersection. i'm not sure why you are bringing that up and what you are hoping would be accomplished. is this a performance concern? do you think it would be more clear converting to sets and then doing the intersection?